specs/waku/v2/waku-relay.md

```diff
@@ -33,26 +34,28 @@ authors: Oskar Thorén <oskar@status.im>, Sanaz Taheri <sanaz@status.im>
+The `relay` protocol supports the following security properties under a static [Adversarial Model](#adversarial-model). Note that data confidentiality[^1], integrity[^2] and authenticity[^3] are currently considered out of the scope of `relay` protocol and must be handled by the upperlevel protocols. 
 
-- **Message Publisher Anonymity**: This property indicates that no adversarial entity can link a published `Message` to its origin i.e., the peer. Note that this feature also implies the unlinkability of the publisher to its published topic ID, this is because  `Message` contains the topic IDs as well.
+[^1]: Confidentiality indicates that an adversary should not be able to learn the data carried by the `relay` protocol.
+[^2]: Integrity indicates that the data transferred by the `relay` protocol can not be tampered with by an adversarial entity without being detected.
+[^3]: Authenticity no adversary can forge data on behalf of a targeted publsiher and make it accepted by other subscribers as if the origin is the target.
 
-- **Topic Subscriber Anonymity:** This feature stands for the inability of any adversarial entity from linking a peer to its subscribed topic IDs.
 
-- **Confidentiality**: The adversary should not be able to learn the data carried by the `relay` protocol.
--  **Integrity**: This feature indicates that the data transferred by the `relay` protocol can not be tampered with by an adversarial entity without being detected.
--  **Authenticity**: No adversary can forge data on behalf of a targeted peer and make it accepted by other peers as if the origin is the target.
-- **Spam resistant**: No adversary can flood the system with spam messages (i.e., publishing a large number of messages in a short amount of time).
+- **Publisher-Message Unlinkability**: This property indicates that no adversarial entity can link a published `Message` to its publsiher. This feature also implies the unlinkability of the publisher to its published topic ID as the `Message` embodies the topic IDs.
+
+- **Subscriber-Topic Unlinkability**: This feature stands for the inability of any adversarial entity from linking a subscriber to its subscribed topic IDs.
+
   
 <!-- TODO: more requirements can be added, but that needs further and deeper investigation-->
 
+
 ### Terminologies
-The term Personally identifiable information (PII) refers to any piece of data that can be used to uniquely identify a user. For example, the signature verification key, and the hash of one's IP address are unique for each user and hence count as PII.
+The term Personally identifiable information (PII) refers to any piece of data that can be used to uniquely identify a user. For example, the signature verification key, and the hash of one's static IP address are unique for each user and hence count as PII.
```

Comment:
In the case of static IP address, there is a one to one mapping between the user and her IP, whereas in the dynamic case, an IP is attributable to multiple users (many to one mapping) therefore an IP cannot uniquely be linked to one single user (it preserves some level of k-anonymity where k is the number of users sharing the same IP). While a dynamic IP address alone does not count PII,  it may result in the linkability risk if combined with other available information.

Reply:
fair!